CoffeeChat/View/MeetupStatusView.swift

```diff
@@ -0,0 +1,335 @@
+        messageTextView.attributedText = unformattedText(for: "\(user.firstName) wants to meet you! Pick a time that works for both of you.")
+    }
+
+    private func setupForWaiting(for user: User) {
+        statusImageView.image = UIImage(named: "reached-out")
+        statusLabel.text = "Reached out"
+        messageTextView.attributedText = unformattedText(for: "Just waiting on \(user.firstName) to pick a time and place!")
+    }
+
+    private func setupForChatScheduled(on date: Date, for user: User) {
+        statusImageView.image = UIImage(named: "scheduled")
+        statusLabel.text = "Chat scheduled"
+
+        let prefix = unformattedText(for: "Get pumped to meet on ")
+        let body = underlinedText(for: "\(formatDate(date))")
+        let suffix = unformattedText(for: ".")
+
+        let fullText = NSMutableAttributedString()
+        fullText.append(prefix)
+        fullText.append(body)
+        fullText.append(suffix)
+
+        if user.instagram != nil || user.facebook != nil {
+            let contactPrefix = unformattedText(for: "\nYou can reach \(user.firstName) on ")
+            let contactLinks = textForSocialMedia(facebook: user.facebook, instagram: user.instagram)
+            let contactSuffix = unformattedText(for: ".")
+            fullText.append(contactPrefix)
+            fullText.append(contactLinks)
+            fullText.append(contactSuffix)
+        }
+
+        messageTextView.attributedText = fullText
+    }
+
+    private func setupForDayBeforeMeeting(on date: Date) {
+        statusImageView.image = UIImage(named: "scheduled")
+        statusLabel.text = "Chatting soon"
+
+        let prefix = unformattedText(for: "Hey! Don't forget to meet at ")
+        let body = underlinedText(for: "\(formatDate(date))")
+        let suffix = unformattedText(for: ".")
+
+        let fullText = NSMutableAttributedString()
+        fullText.append(prefix)
+        fullText.append(body)
+        fullText.append(suffix)
+
+        messageTextView.attributedText = fullText
+    }
+
+    private func setupForNoResponses() {
+        statusImageView.image = UIImage(named: "new-pear")
+        statusLabel.text = "New Pear"
+        messageTextView.attributedText = unformattedText(for: "Uh-oh, looks like neither of you has reached out yet. Be bold and make the first move!")
+    }
+
+    private func setupForChatFinished() {
+        statusImageView.image = UIImage(named: "finished")
+        statusLabel.text = "Finished chat"
+        messageTextView.attributedText = unformattedText(for: "Hope your chat went well! Now you have one more friend at Cornell 😊")
+    }
+
+    private func setupForChatCancelled(with user: User) {
+        statusImageView.image = UIImage(named: "Cancelled")
+        statusLabel.text = "Cancelled"
+
+        let cancelledText = unformattedText(for: "Oh no, it looks like your schedules don’t line up 😢 I hope it works out next time!")
+
+        if user.instagram != nil || user.facebook != nil {
```

Comment:
I think we should probably talk to Cindy about if every user should just always put in their social media.. seems annoying if like one or the other or both are nil.. makes the app a bit inconsistent too...